### ISSUE TITLE
confirming bad accuracy of alexnet on cifar

### ISSUE THREAD

**Issue by kirk86** [IvoryCandy/pytorch-cifar10#5]
I just tried alexnet on cifar and got 69.09% on test set even with cv, although I trained for 100 epochs standard lr for adam (i.e. no mulstistep lr), batch size of 1024 for training and 64 for final test set evaluation.
